tree.go

```diff
@@ -795,6 +795,7 @@ func (ns nodes) findEdge(label byte) *node {
 }
 
 // Route describes the details of a routing handler.
+// Handlers map key is an HTTP method
```

Comment:
The comment doesn't necessarily make it clear though

Reply:
Would `// Handlers map key is an HTTP method, e.g. "get"` be clearer?